relay/server.go

```diff
@@ -124,8 +130,8 @@ func NewServer(
 	blobStore *blobstore.BlobStore,
 	chunkReader chunkstore.ChunkReader,
 	chainReader core.Reader,
-	ics core.IndexedChainState,
-) (*Server, error) {
```

Comment:
nit: not sure if there's any explicit recommendations on this, but I find this format more readable 😅

Reply:
reverted format change